winter_openapi/__init__.py

```diff
@@ -1,3 +1,57 @@
+from openapi_pydantic import Encoding as Encoding310
+from openapi_pydantic import Example as Example310
+from openapi_pydantic import ExternalDocumentation as ExternalDocumentation310
+from openapi_pydantic import Header as Header310
+from openapi_pydantic import Info as Info310
+from openapi_pydantic import License as License310
+from openapi_pydantic import Link as Link310
+from openapi_pydantic import MediaType as MediaType310
+from openapi_pydantic import OAuthFlow as OAuthFlow310
+from openapi_pydantic import OAuthFlows as OAuthFlows310
+from openapi_pydantic import OpenAPI as OpenAPI310
+from openapi_pydantic import Operation as Operation310
+from openapi_pydantic import Parameter as Parameter310
+from openapi_pydantic import PathItem as PathItem310
+from openapi_pydantic import Reference as Reference310
+from openapi_pydantic import RequestBody as RequestBody310
+from openapi_pydantic import Response as Response310
+from openapi_pydantic import Schema as Schema310
+from openapi_pydantic import SecurityScheme as SecurityScheme310
+from openapi_pydantic import Server as Server310
+from openapi_pydantic import ServerVariable as ServerVariable310
+from openapi_pydantic import Tag as Tag310
+from openapi_pydantic import XML as XML310
+from openapi_pydantic.v3.v3_0_3 import Components as Components303
+from openapi_pydantic.v3.v3_0_3 import Contact as Contact303
+from openapi_pydantic.v3.v3_0_3 import Discriminator as Discriminator303
+from openapi_pydantic.v3.v3_0_3 import Encoding as Encoding303
+from openapi_pydantic.v3.v3_0_3 import Example as Example303
+from openapi_pydantic.v3.v3_0_3 import ExternalDocumentation as ExternalDocumentation303
+from openapi_pydantic.v3.v3_0_3 import Header as Header303
+from openapi_pydantic.v3.v3_0_3 import Info as Info303
+from openapi_pydantic.v3.v3_0_3 import License as License303
+from openapi_pydantic.v3.v3_0_3 import Link as Link303
+from openapi_pydantic.v3.v3_0_3 import MediaType as MediaType303
+from openapi_pydantic.v3.v3_0_3 import OAuthFlow as OAuthFlow303
+from openapi_pydantic.v3.v3_0_3 import OAuthFlows as OAuthFlows303
+from openapi_pydantic.v3.v3_0_3 import OpenAPI as OpenAPI303
+from openapi_pydantic.v3.v3_0_3 import Operation as Operation303
+from openapi_pydantic.v3.v3_0_3 import Parameter as Parameter303
+from openapi_pydantic.v3.v3_0_3 import PathItem as PathItem303
+from openapi_pydantic.v3.v3_0_3 import Reference as Reference303
+from openapi_pydantic.v3.v3_0_3 import RequestBody as RequestBody303
+from openapi_pydantic.v3.v3_0_3 import Response as Response303
+from openapi_pydantic.v3.v3_0_3 import Schema as Schema303
+from openapi_pydantic.v3.v3_0_3 import SecurityScheme as SecurityScheme303
+from openapi_pydantic.v3.v3_0_3 import Server as Server303
+from openapi_pydantic.v3.v3_0_3 import ServerVariable as ServerVariable303
+from openapi_pydantic.v3.v3_0_3 import Tag as Tag303
+from openapi_pydantic.v3.v3_0_3 import XML as XML303
```

Comment:
A shorter alternative: import module instead of each separate class.

```
import openapi_pydantic.v3.v3_0_3
openapi_pydantic.v3.v3_0_3.Tag.Config.extra = Extra.ignore
```